backend/repository/bets.go

```diff
@@ -0,0 +1,48 @@
+package repository
+
+type BetsRepository struct {
+	db Database
+}
+
+func NewBetsRepository(db Database) *BetsRepository {
+	return &BetsRepository{db: db}
+}
+
+func (repo *BetsRepository) FirstTimeBets() (int64, error) {
```

Comment:
I'm not really clear on what this is supposed to return/do? Some of my follow on comments are trying to figure that out. 

Can you write a comment explaining what this function is supposed to do?

Reply:
Good question.  `The FirstTimeBets()` function is meant to quickly and efficiently count all first time bets on a particular market.

This number, an `int64` is then multiplied by the https://github.com/openpredictionmarkets/socialpredict/blob/main/backend/setup/setup.yaml#L16 ... `initialBetFee` amount  to calculate the total number of first time bet fees obtained from the market.

This can be used either in payment to the market maker upon resolution of the market, or within the sitewide stats calculation api which helps ensure that no untracked money is created in the system.